code/modules/assembly/flash.dm

```diff
@@ -90,7 +90,7 @@
 		if(M.flash_act(1, 1))
 			M.confused += power
 			terrible_conversion_proc(M, user)
-			M.Stun(1)
+			M.Weaken(2)

```

Comment:
a weaken 2 is weak as fuck, barely a disarm push

flashes used to be a weaken(5)